wgpu-hal/src/gles/egl.rs

```diff
@@ -329,7 +416,7 @@ impl Inner {
         // Testing if context can be binded without surface
         // and creating dummy pbuffer surface if not.
         let pbuffer =
-            if version < (1, 5) || !display_extensions.contains("EGL_KHR_surfaceless_context") {
+            if version < (1, 5) && !display_extensions.contains("EGL_KHR_surfaceless_context") {
```

Comment:
uh oh, I think this would look cleaner by reversing the condition entirely
nice that you are fixing this!